.github/workflows/master.yml

```diff
@@ -26,7 +23,7 @@ on:
         default: ""
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  group: ${{ github.workflow }}-${{ github.run_number || github.ref }}
```

Comment:
@dbczumar Can we also apply https://github.com/mlflow/mlflow/pull/6257#discussion_r923924367 to this line?